Source/Fuse.Models/Fuse.Models.unoproj

```diff
@@ -0,0 +1,33 @@
+{
+  "Copyright": "Copyright (c) Fusetools AS 2017",
+  "Description": "Native Location",
+  "Publisher": "Fusetools AS",
+  "Version": "1.4.0-rc4",
```

Comment:
This version number is incorrect. You can use 0.1.0 as a placeholder until we do a real release.